projects/helper/coreAssets.json

```diff
@@ -2260,7 +2260,9 @@
     "USDT": "0x7277cc818e3f3ffbb169c6da9cc77fc2d2a34895",
     "USDC": "0x836d275563bab5e93fd6ca62a95db7065da94342",
     "WETH": "0xa572bf655f61930b6f0d4546a67cd1376220081a",
-    "WBTC": "0xac0313f97398b585f23f8e50952f10d62350697c"
+    "WBTC": "0xac0313f97398b585f23f8e50952f10d62350697c",
+    "dEDU": "0x597FFfA69e133Ee9b310bA13734782605C3549b7",
+    "GAINZ": "0x32eDd6f3453f4b1F7Ad9DC4CEAF3Cff861f1080F"
```

Comment:
please revert the change to this file and hardcode the addresses inside your adapter